components/blog-index/server.css

```diff
@@ -59,6 +59,10 @@
   display: grid;
   grid-template-columns: 1fr 1fr;
   column-gap: 2rem;
+
+  @media (--screen-large-and-narrower) {
```

Comment:
Maybe should be medium and narrower:

<img width="1156" height="550" alt="image" src="https://github.com/user-attachments/assets/5bead2b1-bedf-4da0-a884-abac9a7bb4c5" />

```suggestion
  @media (--screen-medium-and-narrower) {
```